docs/layouts/shortcodes/generated/mongodb_sync_table.html

```diff
@@ -47,7 +47,7 @@
     </tr>
     <tr>
         <td><h5>--mongodb_conf</h5></td>
-        <td>The configuration for Flink CDC MongoDB sources. Each configuration should be specified in the format "key=value". hosts, username, password, database and collection are required configurations, others are optional. See its <a href="https://nightlies.apache.org/flink/flink-cdc-docs-release-3.0/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc/#connector-options">document</a> for a complete list of configurations.</td>
+        <td>The configuration for Flink CDC MongoDB sources. Each configuration should be specified in the format "key=value". hosts, username, password, database and collection are required configurations, others are optional. See its <a href="https://nightlies.apache.org/flink/flink-cdc-docs-release-3.1/docs/connectors/flink-sources/postgres-cdc/#available-metadata">document</a> for a complete list of configurations.</td>
```

Comment:
It is a mongodb configuration options, why change it to postgres cdc related document address?

Reply:
Thanks for your review, Wrong update, Corrected.